app.rb

```diff
@@ -1,11 +1,23 @@
-require_relative 'book'
+require_relative 'parser'
 
 class App
   attr_accessor :books, :games, :music_albums
 
+  include Parser
+
   def initialize
     @books = []
     @games = []
     @music_albums = []
   end
+
+  def list_all_books
+    return 'No books found' if @books.empty?
+
+    puts @books
+    @books.each do |book|
+      print "\n[BOOK][ID][#{book.id}] Publisher: #{book.publisher},"
+      print "Publish date: #{book.publish_date}, Cover state: #{book.cover_state}"
+    end
```

Comment:
I can see in the above lines of code that you are printing out book items but how come the printed items are appreeating correctly? I was expecting to see the publisher, date, and cover information printed out not `#<Book:0x00007f622f67ae60>`

![image](https://user-images.githubusercontent.com/8858461/191693165-703a6ba7-0200-42d7-ac98-0b4cab4d15c6.png)